streams-tests/jvm/src/test/scala/zio/stream/ChunkSpec.scala

```diff
@@ -84,6 +86,18 @@ object ChunkSpec
             assert(c.toArray.toSeq, equalTo(c.toSeq))
           }
         },
+        test("toArray for an empty Chunk of type String") {
+          assert(Chunk.empty.toArray[String], equalTo(Array.empty[String]))
+        },
+        testM("toArray with elements of type String") {
+          check(chunkGen(stringGen)) { c =>
+            assert(c.toArray.toSeq, equalTo(c.toSeq))
+          }
+        },
+        test("toArray for a Chunk of a given type") {
+          val v: Vector[Any] = Vector("String", 1, Value(2))
+          assert(Chunk.fromIterable(v).toArray.toVector, equalTo(v))
+        },
```

Comment:
Could  you add  @Jentsch 's testcase? `Chunk(1).filter(_ == 2).map(_.toString).toArray[String]`

Reply:
Sure

Reply:
it works but sounds there is a problem in circleci will check